azure-iot-device/azure/iot/device/iothub/config.py

```diff
@@ -0,0 +1,29 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import logging
+from azure.iot.device.common.config import BasePipelineConfig
+
+logger = logging.getLogger(__name__)
+
+
+class IoTHubPipelineConfig(BasePipelineConfig):
+    """A base class for storing all configurations/options shared across the Azure IoT Python Device Client Library.
+    More specific configurations such as those that only apply to the IoT Hub Client will be found in the respective
+    config files.
+    """
+
+    super
```

Comment:
>super [](start = 4, length = 5)

?

Reply:
removed.

---
In reply to: [337672538](https://github.com/Azure/azure-iot-sdk-python/pull/368#discussion_r337672538) [](ancestors = 337672538)